projects/SSTorytime/services/sstorytime/module.nix

```diff
@@ -167,11 +168,5 @@ in
         host ${cfg.database.dbname} ${cfg.database.user} localhost trust
```

Comment:
@eljamm I think this is meant for that?

Edit: Maybe not.

Not many services are considering that possibilty of having a different name, as ensureDBOwnership description is this.

 > Grants the user ownership to a database with the same name.
 This database must be defined manually in
 `[](#opt-services.postgresql.ensureDatabases).`

Reply:
Yes, most services assume `dbname` and `user` will be the same, which doesn't seem right to me.